numpy/lib/function_base.py

```diff
@@ -241,6 +241,39 @@ def iterable(y):
     return True
 
 
+def _hist_bin_ise(x):
+    """
+    Histogram bin estimator based on minimizing the estimated integrated squared error (ISE).
+
+    The number of bins is chosen by minimizing the estimated ISE against the unknown true distribution.
+    The ISE is estimated using cross-validation and can be regarded as a generalization of Scott's rule.
```

Comment:
If this is based off Scott's rule, then it would make more sense further down in the file below that rule